### PR TITLE
update notebook

### PR DESCRIPTION
**Pull request by foleem20**
Remove O6 and update rates table

### CODE REVIEW

File: plots-and-tables.ipynb [hunk 1]
```diff
@@ -846,7 +846,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.11"
+   "version": "3.7.2"
```
Comment (lpsinger) [inline plots-and-tables.ipynb:849]: This Poetry project requires Python >= 3.8: https://github.com/lpsinger/observing-scenarios-simulations/blob/main/pyproject.toml#L10

**Review by lpsinger — CHANGES_REQUESTED** [review]
This PR must be missing some pieces:

1. [ ] Where is the code to pull apart the runs with the Farah distribution into BNS, NSBH, and BBH subpopulations?
2. [ ] Since we are using the Farah distribution for all three populations, it would seem that we should normalize all three subpopulations to a one integrated astrophysical rate that spans the whole parameter space.

File: plots-and-tables.ipynb (hunk 1 above)
Comment (weizmannk) [reply plots-and-tables.ipynb:849]: Hi Leo,
We would like to make a new simulation of the popualtions farah, in its own right.
That is to say, I split from the file farah.h5, so as to obtain the populations farah_bbh, farah_bns and farah_nsbh.

Comment (weizmannk) [reply plots-and-tables.ipynb:849]: Tomorrow I will make a report  about the process and  the result.

**Pull request closed without merging**